data/json/items/armor/integrated.json

```
@@ -173,6 +173,12 @@
         "covers": [ "hand_l", "hand_r", "foot_l", "foot_r", "leg_l", "leg_r", "arm_l", "arm_r", "torso", "head" ],
         "coverage": 100,
         "encumbrance": 2
+      },
+	  {
```

```suggestion
      {
```
Just a minor linting issue